Mage.Sets/src/mage/cards/c/ConsumingBlob.java

```diff
@@ -1,23 +1,17 @@
 package mage.cards.c;
 
 import mage.MageInt;
-import mage.MageObject;
```

Comment:
I see your point. Should unused imports be cleaned up later, or just left in?

Reply:
It will be clean someday on another smaller edit. Also I don't recommends to make massive source code reformat/clean PR -- there are many active PRs that can be broken after that.

Simple hint for legacy code edit:
 - if you fix 1-2 lines then don't use auto-format;
 - if you rework or add multiple blocks of code then auto-format it.